test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	if [ -z "$TRAVIS" ]; then
+		skip "This test is intended to run only in Travis (use TRAVIS=true to run it anyway)..."
+	fi
+
+	create_test_environment "$TEST_NAME"
+
+}
+
+test_teardown() {
+
+	if [ -z "$TRAVIS" ]; then
+		return
+	fi
+
```

Comment:
you are never cleaning the tea environment , if I run the test with TRAVIS=true the test env will be left behind

Reply:
Ahh Yes..great catch..neeed to add a destroy

Reply:
Done